internal/impl/confluent/processor_schema_registry_encode_test.go

```diff
@@ -210,7 +210,7 @@ func TestSchemaRegistryEncodeAvro(t *testing.T) {
 		{
 			name:        "message doesnt match schema",
 			input:       `{"Address":{"my.namespace.com.address":"not this","Name":"foo"}}`,
-			errContains: "schema does not specify default value",
+			errContains: "cannot decode textual union: cannot decode textual record",
```

Comment:
Changed as was getting this new error:

```
--- FAIL: TestSchemaRegistryEncodeAvro (0.00s)
    --- FAIL: TestSchemaRegistryEncodeAvro/message_doesnt_match_schema (0.00s)
        processor_schema_registry_encode_test.go:231:
            	Error Trace:	processor_schema_registry_encode_test.go:231
            	Error:      	"cannot decode textual record \"foo.namespace.com.identity\": cannot decode textual union: cannot decode textual record \"my.namespace.com.address\": expected: '{'; actual: '\"' for key: \"my.namespace.com.address\" for key: \"Address\"" does not contain "schema does not specify default value"
            	Test:       	TestSchemaRegistryEncodeAvro/message_doesnt_match_schema
```

I am assuming because a different codec parser is being used the error message is different.